lib/pharos/config.rb

```diff
@@ -122,7 +127,7 @@ def kube_client(kubeconfig)
         api_port = 6443
       else
         api_address = 'localhost'
-        api_port = master_host.bastion.host.ssh.gateway(master_host.api_address, 6443)
+        api_port = master_host.bastion.host.ssh.gateway.open(master_host.api_address, 6443)
```

Comment:
`ssh.gateway` now returns a `Net::SSH::Gateway` instead of directly doing `gateway.open`.